nginx_vts_exporter.go

```diff
@@ -144,7 +150,7 @@ type Exporter struct {
 	serverMetrics, upstreamMetrics, cacheMetrics map[string]*prometheus.GaugeVec
 }
 
-func newServerMetric(metricName string, docString string, labels []string) *prometheus.GaugeVec {
+func newServerMetric(metricName string, docString string, labels []string, namespace string) *prometheus.GaugeVec {
```

Comment:
Well we can set the global variable `namespace` from Env variable so we can use it anywhere without changing any function header.